kratos/solving_strategies/schemes/residual_based_bossak_displacement_scheme.hpp

```diff
@@ -219,23 +219,20 @@ class ResidualBasedBossakDisplacementScheme
         const int num_nodes = static_cast<int>( rModelPart.Nodes().size() );
         const auto it_node_begin = rModelPart.Nodes().begin();
 
-        #pragma omp parallel for
-        for(int i = 0;  i < num_nodes; ++i) {
-            auto it_node = it_node_begin + i;
+        IndexPartition<std::size_t>(num_nodes).for_each(array_1d<double,3>(), [&](std::size_t Index, array_1d<double,3>& rDeltaDisplacement){
```

Comment:
Including `TLS` in the variable used for the tls would help to keep the overview more

Reply:
I normally do it in that way (I don't know why I decided to keep the old name in here :confused:).

Reply:
Done.